File: Python/Sorting/bubblesort.py
def bubblesort(arr):
    length = len(arr)

    swap = True
    while swap:
        swap = False
        for x in range(length-1):
            if arr[x] > arr[x+1]:
                [arr[x], arr[x+1]] = [arr[x+1], arr[x]]
                swap = True

    return arr


    return arr;


arr = [5, 10, 1, 9, 2, 8, 3, 7, 4, 6, 1];
print(bubblesort(arr));

File: Javascript/bubblesort.js

const bubblesort = (arr) => {
  let swap = false;
  let length = arr.length;
  do {
    swap = false;
    for (let i = 0; i < length-1; i++) {
      if(arr[i] > arr[i+1]) {
          [arr[i], arr[i+1]] = [arr[i+1], arr[i]];
          swap = true;
      }
    }
  } while (swap);

  return arr;
}


let arr = [5, 10, 1, 9, 2, 8, 3, 7, 4, 6, 1];
console.log(bubblesort(arr));
